backend/api/Controllers/RobotController.cs

```diff
@@ -654,6 +657,99 @@ public async Task<ActionResult> ResumeMission([FromRoute] string robotId)
         return Ok(mission);
     }
 
+
+    /// <summary>
+    /// Starts a mission which drives the robot to the nearest safe position
+    /// </summary>
+    /// <remarks>
+    /// <para> This query starts a localization for a given robot </para>
+    /// </remarks>
+    [HttpPost]
+    [Route("{robotId}/{asset}/{deck}/safe-position")]
```

Comment:
```suggestion
    [Route("{robotId}/{asset}/{deck}/go-to-safe-position")]
```
I think the endpoint name should reflect the action it is provoking

Reply:
Agreed